packages/angular/angular.json

```diff
@@ -0,0 +1,28 @@
+{
+  "$schema": "../.././node_modules/@angular/cli/lib/config/schema.json",
+  "version": 1,
+  "newProjectRoot": "projects",
+  "projects": {
+    "sentry-angular": {
+      "projectType": "library",
+      "root": ".",
+      "sourceRoot": "src",
+      "prefix": "lib",
```

Comment:
this prefix still apply?

Reply:
Good catch, `prefix` is an automatically generated property that tells the Angular CLI which prefix to add to the selector of a new component/directive/pipe when creating one via the CLI. Since we're not using prefixes in our directive selector (the selector is just `trace` instead of e.g. `lib-trace`), we can simply remove the property.